contracts/src/TokenSpoke/TokenSpoke.sol

```diff
@@ -320,6 +318,16 @@ abstract contract TokenSpoke is ITokenSpoke, TeleporterOwnerUpgradeable, SendRee
         uint256 amount
     ) internal virtual;
 
+    /**
+     * @notice Handles fees sent to this contract
+     * @param feeTokenAddress The address of the fee token
+     * @param feeAmount The amount of the fee
+     */
```

Comment:
Since this is a virtual function, can we make the comment on what it's supposed to do much more explicit? So anyone wanting to override it knows exactly what it should do.

Reply:
updated